sdk/cargo-build-sbf/src/main.rs

```diff
@@ -135,11 +135,31 @@ pub fn is_version_string(arg: &str) -> Result<(), String> {
     Err("a version string may start with 'v' and contains major and minor version numbers separated by a dot, e.g. v1.32 or 1.32".to_string())
 }
 
+fn get_home_dir() -> PathBuf {
```

Comment:
nit: `get_` is not idiomatically rust, and `home_dir()` is preferred

Reply:
Oh didn't, know that I based it off `get_latest_platform_tools_version()` and `get_base_rust_version()`.
Pushed a fix.

Reply:
Convention isn't always followed unfortunately :sweat_smile: